### PR TITLE
add SquareArea in Javascript

### PR DESCRIPTION
**Pull request by azriakmalk**

### CODE REVIEW

File: src/javascript/azriakmalk-SquareArea.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//the function is count square area
+const Square = (n) => {
```
Comment (jazibjafri) [inline src/javascript/azriakmalk-SquareArea.js]: @azriakmalk rename the function to be as same as in the filename, or vice versa.

**Review by jazibjafri — CHANGES_REQUESTED** [review]
@azriakmalk

File: src/javascript/azriakmalk-SquareArea.js (hunk 1 above)
Comment (azriakmalk) [reply src/javascript/azriakmalk-SquareArea.js]: @JazibJafri okey

File: src/javascript/azriakmalk-CountSquareArea.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+//the function is count square area
+const Square = (n) => {
```
Comment (jazibjafri) [inline src/javascript/azriakmalk-CountSquareArea.js]: @azriakmalk rename the function. Checkout other examples in the src folder

**Review by jazibjafri — CHANGES_REQUESTED** [review]

File: src/javascript/azriakmalk-CountSquareArea.js (hunk 2 above)
Comment (azriakmalk) [reply src/javascript/azriakmalk-CountSquareArea.js]: @JazibJafri okey

**Review by jazibjafri — APPROVED** [review]

**Pull request merged**